style.css

```diff
@@ -0,0 +1,1580 @@
+  }
+
+  .details-card:hover {
+    cursor: pointer;
+  }
+
+  .project-image1 {
+    display: flex;
+    width: 100%;
+    height: 220px;
+    border-radius: 8px;
+    background-image: url('./images/project_1.svg');
+  }
+
+  .project-image2 {
+    display: flex;
+    width: 100%;
+    height: 220px;
+    border-radius: 8px;
+    background-image: url('./images/project2.svg');
+  }
+
+  .project-image3 {
+    display: flex;
+    width: 100%;
+    height: 220px;
+    border-radius: 8px;
+    background-image: url('./images/project3.svg');
+  }
+
+  .project-image4 {
+    display: flex;
+    width: 100%;
+    height: 220px;
+    border-radius: 8px;
+    background-image: url('./images/project4.svg');
+  }
+
+  .card-info {
+    width: 100%;
+  }
+
+  .project-description {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    width: 100%;
+    height: 288px;
+  }
+
+  .project-details {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    height: 120px;
+    margin-top: 10px;
+  }
+
+  .project-title {
+    width: 60%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .client-info {
+    display: flex;
+    margin-bottom: 12px;
+    padding: 0;
+    align-items: center;
+    height: 10px;
+    width: 100%;
+  }
+
+  .Cinf {
+    display: flex;
+    position: relative;
+    justify-content: flex-start;
+    margin: 0;
+    padding: 0;
+    height: 10px;
+    align-items: center;
+  }
+
+  .Cinf li {
+    margin-left: 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    padding-right: 12px;
+    color: #7a869a;
+  }
+
+  .canopy {
+    list-style: none;
+    width: 55px;
+    height: 16px;
+    position: relative;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #344563;
+  }
+
+  .client-name {
+    width: 55px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #344563;
+    margin-bottom: 20px;
+  }
+
+  .project-info {
+    width: 100%;
+    height: 76px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .pl {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 53px;
+    height: 24px;
+    border-radius: 8px;
+    padding-left: 0;
+  }
+
+  .pl li {
+    margin-left: 0;
+    margin-right: 12px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 89px;
+    height: 24px;
+    color: #396df2;
```

Comment:
[OPTIONAL] I have observed that you are having issues regarding your pull request. But kindly implement the suggestion given below by previous reviewers.

As suggested by the previous reviewer [here](https://github.com/fredo509/Portfolio_setup_mobile_first/pull/8#discussion_r1115222343), the correct color is color: #6070FF;. Let's update this to improve our design.